MaterialDesignThemes.Wpf/DataGridTextColumn.cs

```diff
@@ -15,6 +17,30 @@ public class DataGridTextColumn : System.Windows.Controls.DataGridTextColumn
             return null;
         }
 
+        protected override FrameworkElement? GenerateElement(DataGridCell cell, object dataItem)
+        {
+            var element = base.GenerateElement(cell, dataItem);
+
+            if (
+                (cell.Column as MaterialDesignThemes.Wpf.DataGridTextColumn)?.ElementStringFormat is string stringFormat
+                && element is TextBlock t
+                && t.GetBindingExpression(TextBlock.TextProperty)?.ParentBinding is Binding binding
+                )
+            {
+                var NewBinding = binding.Clone();
```

Comment:
Consider using camel case rather than pascal case for local variables. e.g. "newBinding" or "bindingClone" or similar.